internal/tx/tx.go

```diff
@@ -168,20 +191,31 @@ func createTx(blockData common.BlockData, nonce [16]byte) ([]byte, error) {
 
 	apollob = apollob.AddLoadedUTxOs(utxos...)
 	apollob = apollob.
-		PayToContract(contractAddress, &postDatum, int(validatorOutRef.Output.Lovelace()), true, apollo.NewUnit(validatorHash, "lord tuna", 1)).
-		SetTtl(int64(currentTipSlotNumber+180000)).
-		PayToAddress(myAddress, 2000000, apollo.NewUnit(validatorHash, "TUNA", 5000000000)).
-		SetValidityStart(int64(currentTipSlotNumber)).MintAssetsWithRedeemer(
-		apollo.NewUnit(validatorHash, "TUNA", 5000000000),
-		Redeemer.Redeemer{
-			Tag:   Redeemer.MINT,
-			Index: 0,
-			Data: PlutusData.PlutusData{
-				PlutusDataType: PlutusData.PlutusArray,
-				TagNr:          121,
-				Value:          PlutusData.PlutusIndefArray{},
+		PayToContract(
+			contractAddress, &postDatum, int(validatorOutRef.Output.Lovelace()), true, apollo.NewUnit(validatorHash, "lord tuna", 1),
+		).
+		SetValidityStart(int64(currentTipSlotNumber)).
+		SetTtl(int64(currentTipSlotNumber+500)).
+		PayToAddress(
+			myAddress, 2000000, apollo.NewUnit(validatorHash, "TUNA", 5000000000),
```

Comment:
This 2000000 lovelace is always 1172320 in mainnet. It's 1051640 on preview. We should probably capture the amount of lovelace from the input and use the exact same amount in the output.


NewUnit returns a Unit, which is:
```go
type Unit struct {
	PolicyId string
	Name     string
	Quantity int
}
```

I think validatorHash needs to be TUNA's Policy ID here and below.